Swiftfin/Views/Representable/FontPicker.swift

```diff
@@ -0,0 +1,61 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2022 Jellyfin & Jellyfin Contributors
+//
+
+import SwiftUI
+import UIKit
+
+// https://github.com/SwapnanilDhol/SUIFontPicker
+public struct FontPicker: UIViewControllerRepresentable {
```

Comment:
I agree with the other parts and will fix them.

However, since FontPickerView is literally a picker, I think it's better to process it with the font information selected from the outside(ex: `SettingsView`) rather than setting the `Defaults`'s value inside
so I will keep 'onFontPick: (UIFontDescriptor) -> Void'.